src/survey/controllers/create_survey_controller.py

```diff
@@ -1,49 +1,75 @@
+import json
+
 from api.utils import failure_response
 from api.utils import modify_attribute
 from api.utils import success_response
 from match import match_status
 from match.models import Match
-from person.models import Person
 from rest_framework import status
-from survey.constants import RATINGS
+from survey import constants
 from survey.models import Survey
 
 
 # TODO(@chalo2000) Convert to celery task
 class CreateSurveyController:
-    def __init__(self, request, data):
+    def __init__(self, request, data, match_id):
         self._request = request
         self._data = data
+        self._match_id = match_id
 
     def process(self):
```

Comment:
im glad you guys are using controllers and using `process` :))) this is an instagram coding standard <3